models/playerModel.js

```diff
@@ -5,7 +5,7 @@ const playerSchema = new mongoose.Schema({
     shifts: [[String]],
     shiftsTotal: Number,
     shiftsTimer: Number,
-    health: [Number],
+    health: [String],
```

Comment:
Since this thing is using mongo (D:), I _think_ this should be okay as an in-place migration as long as the code reading the model can accept either a number or a string?

If the database holds a number, and we load the site to view that user's page, does the JS break due to code that expects a string receiving a number? Or, possibly, due to Mongoose validating its data and rejecting the out-of-date document content?

Reply:
ah, I misread a bit. It should be possible to convert it to a string so that it doesn't break, but I am going to want to get this deployed for Dunk's event run. What I mean is: a number can be converted infallibly to a string, but a string cannot be converted infallibly to a number. So before we deploy to the main site, it'd be good to perform this conversion so that the site continues to work for people who haven't updated their mod